src/TeamSyncer.ts

```diff
@@ -299,6 +300,49 @@ export class TeamSyncer {
         }
     }
 
+    public async syncCustomEmoji(teamId: string, client: WebClient): Promise<void> {
+        // if (!this.getTeamSyncConfig(teamId, 'customEmoji')) {
+        //     log.warn(`Not syncing custom emoji for ${teamId}`);
+        //     return;
+        // }
+        log.info(`Syncing custom emoji ${teamId}`);
+
+        const response = await client.emoji.list();
```

Comment:
We usually create a response type interface because the client sucks and doesn't return any types.